src/config/passport.conf.js

```diff
@@ -131,9 +133,14 @@ export default (passport) => {
         );
       }
 
+      // If the user's email address is not verified yet
+      if (!user.verified) {
+        return done(null, false, { loginMessage: 'Please confirm your email address. <a href="/verify">Send again</a>'})
```

Comment:
What's `/verify`? It's invalid on the server without a token and doesn't exist on the client-side.